src/partisan_peer_service_client.erl

```diff
@@ -61,8 +61,8 @@ init([Peer, From]) ->
     case connect(Peer) of
         {ok, Socket} ->
             {ok, #state{from=From, socket=Socket, peer=Peer}};
-        _Error ->
-            {stop, normal}
+        {error, Reason} ->
```

Comment:
I'm pretty sure this change is going to prevent the simulation scenarios from working, because sometimes it takes several connection attempts before the port is opened.  Was this necessary?